README.md

```diff
@@ -1,249 +1,263 @@
+| [GET]    | /profiles/users/:profile_id <b>(BUG: /profiles/users route does not exist; app.js only connects to /profiles/user)</b> | Returns an array filled with event objects that contains information based on profile_id and role_id.                    |
+| [GET]    | /profile/role/:role_id <b>(BUG: does not return any data)</b>      | Returns an array filled with event objects that contain information based on role_id for all profiles of a role_id type. |
+| [POST]   | /profile/                   | Requires a name, password, and email. Registers a new user.                                                              |
+| [PUT]    | /profile/                   | Returns an event object with the specified `okta`. Updates specific profile.                                             |
+| [DELETE] | /profile/:okta_id/             | Returns an event object with the specified `okta`. Deletes specific profile.                                             |
+#### User:
+<p>These endpoints are user-focused. As opposed to the more flexible Profile endpoints where <b>profile_id</b> must be specified, these endpoints retrieve data specific only to the user profile that is making the API request by using the logged-in user's <b>profile_id</b>.</p>
 
-```
-{
-  id: INCREMENT,
-  user_id: INTEGER,
-}
-```
+| Method | URL              | Description                                                             |
+| ------ | ---------------- | ----------------------------------------------------------------------- |
+| [GET]  | /user/           | Returns an event object with the specified `okta` and `type`.           |
+| [GET]  | /user/inbox/     | Returns an event object with the specified `okta`. <b>(NOT IMPLEMENTED)</b>    |
+| [GET]  | /user/schedules/ | Returns an event object with the specified `okta`.                      |
+| [PUT]  | /user/           | Returns an event object with the specified `id`. Updates specific user. |
 
 <h1>Parents</h1>
 
 ```
 {
-  id: INCREMENT,
-  user_id: INTEGER,
+  parent_id: INCREMENT (primary key, auto-increments, generated by database),
+  profile_id: INTEGER (required, foreign key),
 }
 ```
 
+| Method | URL                    | Description                                                         |
+| ------ | ---------------------- | ------------------------------------------------------------------- |
+| [GET]  | /parent/:profile_id/children/  | Returns an array filled with children event objects with the specified `profile_id`. |
+| [GET]  | /parent/:profile_id/schedules/ | Returns an array filled with schedules event objects with the specified `profile_id`. |
+
 <h1>Children</h1>
 
 ```
 {
-  id: INCREMENT,
-  user_id: INTEGER,
-  parent_id: INTEGER,
-  username: STRING,
-  age: INTEGER,
+  child_id: INCREMENT (primary key, auto-increments, generated by database),
```

Comment:
Same here. Awesome job on renaming the id to be more specific with the user type.